PyInstaller/building/api.py

```diff
@@ -636,13 +637,21 @@ def assemble(self):
             exe = tmpnm
         exe = checkCache(exe, strip=self.strip, upx=self.upx)
         self.copy(exe, outf)
-        if self.append_pkg:
-            logger.info("Appending archive to EXE %s", self.name)
-            self.copy(self.pkg.name, outf)
+        if subprocess.check_call(['objcopy','--help']) != 0:
+            if self.append_pkg:
+                logger.info("Appending archive to EXE %s", self.name)
+                self.copy(self.pkg.name, outf)
+            else:
```

Comment:
@matysek Well, it may be rarely used, but I can' t remember we decided to drop it. (And I'm in favour of keeping it since it is simple (just the branch here) and leads a way to make MERGE simpler and working again.)